quetz/tasks/mirror.py

```diff
@@ -540,37 +699,30 @@ def synchronize_packages(
         logger.error(f"channel {channel_name} not found")
         return
 
-    host = new_channel.mirror_channel_url
+    for mirror_channel_url in new_channel.mirror_channel_urls:
+        remote_repo = RemoteRepository(mirror_channel_url, session)
 
-    remote_repo = RemoteRepository(new_channel.mirror_channel_url, session)
-
-    user_id = auth.assert_user()
+        auth.assert_user()
```

Comment:
Why doe we not need the user ID anymore here?

Reply:
Because it was needed for `create_packages_from_channeldata` (will comment on that below)